src/helpers.js

```diff
@@ -23,5 +27,5 @@ export const stringifyQuery = (params = {}, prefix) => {
     return `${key}=${encodeURIComponent(value)}`;
   });
 
-  return [].concat.apply([], query).join('&');
+  return query.join('&');
```

Comment:
Yeah, this seems a little confusing. It looks like the setter for `.search` adds the leading `?` if you don't provide it, but the getter always returns the full query string (e.g. `?page=2`)